examples/minimal/packages/contracts/mud.config.ts

```diff
@@ -37,6 +37,32 @@ export default defineWorld({
       key: ["owner", "item", "itemVariant"],
     },
   },
+  namespaces: {
+    royale: {
```

Comment:
this "minimal example" is deviating from being a minimal example

is there another place we can put this?

Reply:
Agreed - I'll make another example or maybe put it in some test

Reply:
we have `test/mock-game-contracts`, could add another `test/multiple-namespaces`